### PR TITLE
Add schedule clock icon

### PR DESCRIPTION
**Pull request by jkosir**

### CODE REVIEW

File: src/stateless/icon.vue [hunk 1]
```diff
@@ -4,6 +4,13 @@
             <div class="icon-caret"></div>
         </div>
 
+        <svg v-if="name === 'schedule-clock'" :style="styleOverride" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
+            <g fill-rule="nonzero">
```
Comment (kurent) [inline src/stateless/icon.vue]: When exporting SVG from Sketch, there are always some unneeded residuals like fill-rule="nonzero". To keep SVGs clean you should use [SVGomg](https://jakearchibald.github.io/svgomg/).

File: src/stateless/icon.vue [hunk 2]
```diff
@@ -4,6 +4,13 @@
...
+                <path d="M8 0C3.6 0 0 3.6 0 8s3.6 8 8 8 8-3.6 8-8-3.6-8-8-8zm0 14c-3.3 0-6-2.7-6-6s2.7-6 6-6 6 2.7 6 6-2.7 6-6 6z"/>
```
Comment (kurent) [inline src/stateless/icon.vue]: add class="path" to the SVG so its properly styled from outside

File: src/stateless/icon.vue (hunk 1 above)
Comment (jkosir) [reply src/stateless/icon.vue]: This went through svgomg but it doesn't remove fill-rule. It is redundant in this case so I manually removed it.

File: src/stateless/icon.vue (hunk 2 above)
Comment (jkosir) [reply src/stateless/icon.vue]: fixed

**Pull request merged**